src/streaming/controllers/PlaybackController.js

```diff
@@ -257,7 +257,8 @@ function PlaybackController() {
      */
     function getStreamStartTime(ignoreStartOffset) {
         let presentationStartTime;
-        let startTimeOffset = !ignoreStartOffset ? parseInt(URIQueryAndFragmentModel(context).getInstance().getURIFragmentData().s, 10) : NaN;
+        let fragData = URIQueryAndFragmentModel(context).getInstance().getURIFragmentData();
+        let startTimeOffset = !ignoreStartOffset ? parseInt(fragData.s || fragData.t, 10) : NaN;
```

Comment:
Yeah I thought about the string aspect last night when doing some console testing to make sure JS evals this way.  You are right with the string in parseInt no issue here but I did see same pattern in code for PR #1387 so maybe @LloydW93 can take a look there.